CMakeLists.txt

```diff
@@ -123,28 +124,31 @@ find_library(HIREDIS_LIBRARY
   PATHS
   third_party/hiredis/install/lib)
 
-find_library(ALIVE_LIBRARY alive2 PATHS "${ALIVE_DIR}/build")
+find_library(ALIVE_LIBRARY alive2 PATHS "${ALIVE_DIR}/build" NO_DEFAULT_PATHS)
```

Comment:
I can see cmake documentation for NO_DEFAULT_PATH but not NO_DEFAULT_PATHS, can you point me to some documentation about this?

Reply:
Sorry that was typos.

Reply:
do you think those are needed? do they help?

Reply:
This option forces the find path, this may prevent potential library lookup mistake in the future.